packages/server/package.json

```diff
@@ -62,20 +62,20 @@
     "@opentelemetry/instrumentation-undici": "0.10.1",
     "@opentelemetry/sdk-metrics": "1.30.1",
     "@opentelemetry/sdk-node": "0.57.2",
-    "@smithy/util-retry": "4.0.1",
-    "@smithy/util-stream": "4.1.2",
+    "@smithy/util-retry": "4.0.2",
+    "@smithy/util-stream": "4.2.0",
     "bcryptjs": "3.0.2",
-    "body-parser": "1.20.3",
+    "body-parser": "2.2.0",
     "bowser": "2.11.0",
-    "bullmq": "5.44.3",
+    "bullmq": "5.45.2",
     "bytes": "3.1.2",
     "compression": "1.8.0",
     "cookie-parser": "1.4.7",
     "cors": "2.8.5",
     "cron-validator": "1.3.1",
     "dataloader": "2.2.3",
     "dotenv": "16.4.7",
-    "express": "4.21.2",
+    "express": "5.1.0",
```

Comment:
Looks like express tried to upgrade to v5. I think we'll want to hold this back for now